File: MT.Model/Dictionary/DayOff.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region DayOff:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a DayOff:EntityBase.
    /// </summary>
    public class DayOff : EntityBase
    {
        [Key]
        public Guid DayOffID { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        public int Rownum { get; set; }
    }
    #endregion
}

File: BECore/app/model/DayOff.js
/// <summary>
/// Model EXTJS DayOff.
/// </summary>
Ext.define('QLDT.model.DayOff', {
    extend: 'Ext.data.Model',
    idProperty: 'DayOffID',
    idType: 'Guid',

    fields: [
		{ name: "DayOffID", type: "string" },
		{ name: "Date", type: "date" },
		{ name: "Description", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "Rownum", type: "int" },
    ]
});

